data/dispersion csv files/Valetudo.csv

```diff
@@ -0,0 +1,31 @@
+﻿attribute_class; parameter_name; mean_value; standard_deviation;
```

Comment:
Is this file crucial for this release? If this is not the case, consider deleting it to prevent accumulating unnecessary files.

Reply:
Furthermore, the standard for CSV files is to be comma delimited. So every ";" should be replaced by ",".

Reply:
We need this file to run the `test_create_dispersion_dictionary` function. It is not a duplicated file, neither a large one, so I would say it is not a big deal having it here.

Regarding the semicolon problem, well..., it was a workaround since there're some items that present commas, as you can see at the ensembleMember, which contains a list.
I think the best way would be having these information as .json files. However, it is not so human-readable as the csv one, so I don't know. I rather take few hours to think what is the best solution here.

Reply:
YAML files would be the best option in my opinion. I'll raise an issue regarding this, but the implementation works, so I do not believe we should hold back this PR.

However, I do believe the docstring should give a better explanation of why you should use ";".

Reply:
The idea of using csv files was to have an easy interface, allowing the user to write different csv files from excel, for example, and running a batch of simulations with them. YAML does not allow that.

Anyway, I agree we should not spend too many time with this method. It was created mainly to solve a specific problem last year, which was already addressed,